_woc/Welcome to my WoC blog.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Text Sentiment Analysis Starter Blog"
+excerpt: "This is the inital blog post to derive a generalized interest for the project of Text Sentiment Analysis."
+toc: true
+toc_sticky: true
+toc_label: "Contents"
+
+categories:
+  - woc
+
+tags:
+  - Winter of Code
+  - Developers
+  - Developer Student Clubs
+  - DSC NSEC
+  - DSC IEM
+
+
+og_image: /assets/images/WoC2020logo.jpg
```

Comment:
Change the header images to your own favourite image. Make the website stand out and not dull. :)
All images go under assets/images . https://github.com/khanfarhan10/khanfarhan10.github.io/tree/master/assets/images